static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx

```diff
@@ -1,33 +1,41 @@
-import {getMeta} from 'sentry/components/events/meta/metaProxy';
-import {KeyValueListData} from 'sentry/types';
+import {Event, KeyValueListData} from 'sentry/types';
 import {defined} from 'sentry/utils';
 
-import getRuntimeKnownDataDetails from './getRuntimeKnownDataDetails';
-import {RuntimeData, RuntimeKnownDataType} from './types';
+import {getRuntimeKnownDataDetails} from './getRuntimeKnownDataDetails';
+import {runtimeKnownDataValues} from './index';
+import {RuntimeData} from './types';
 
-function getRuntimeKnownData(
-  data: RuntimeData,
-  runTimerKnownDataValues: Array<RuntimeKnownDataType>
-): KeyValueListData {
+type Props = {
+  data: RuntimeData;
+  meta: NonNullable<Event['_meta']>['runtime'];
+};
+
+export function getRuntimeKnownData({data, meta}: Props): KeyValueListData {
   const knownData: KeyValueListData = [];
 
-  const dataKeys = runTimerKnownDataValues.filter(runTimerKnownDataValue =>
-    defined(data[runTimerKnownDataValue])
-  );
+  const dataKeys = runtimeKnownDataValues.filter(runTimerKnownDataValue => {
+    if (!defined(data[runTimerKnownDataValue])) {
+      if (meta[runTimerKnownDataValue]) {
+        return true;
+      }
+      return false;
```

Comment:
IMO this is a bit more readable
```suggestion
      return !!meta[runTimerKnownDataValue];
```
or
```
      return meta[runTimerKnownDataValue] ? true : false;
```

Reply:
I have copied and paste this logic in many other contexts but I can update all of them in a separate PR 😉

Reply:
in this case the second suggestion is more readable IMO but usually I prefer not use ternaries as it can be difficult to read imo